lms/djangoapps/certificates/api.py

```diff
@@ -27,7 +27,7 @@
     CertificateTemplate,
 )
 from certificates.queue import XQueueCertInterface
-
+from branding import api as branding_api, get_logo_url
```

Comment:
This import is confusing to me -- why are we importing get_logo_url specifically instead of just referencing it as part of branding_api?